assembly/vm-mock/env.ts

```diff
@@ -19,6 +19,9 @@
 @external("massa", "assembly_script_mock_call")
 export declare function mockScCall(value: StaticArray<u8>): void;
 
+@external("massa", "assembly_script_set_chain_id")
+export declare function mockSetChainId(value: number): void;
+
```

Comment:
i dont see real usecase where this would be needed.. 
i also don't see any usecase where chainId would be needed in a smart contract. It would mean you are developing a SC compatible with multiple chains ? and do some stuff differently on purpose?
sounds weird to me. and just not optimized in term of gas use

Reply:
Used by Umbrella SC for instance, closing this...